README.md

```diff
@@ -40,12 +42,30 @@ Without any prior setup on your local site, you can edit files directly on githu
 Once you click on it, you are able to edit and preview your changes. Saving is equal to making a commit.
 
 If you want, you can also upload a picture to 
-[_pages/Figures/img_name.png](_pages\Figures) 
+[_pages/Figures/img_name.png](_pages/Figures) 
 of yours and reference it in your page:
 
 ![Upload your Image](upload_your_image.png)
 
-> you commit your changes to the branch you selected!
+> you commit (=save) your changes to the branch you selected!
+
+### VS Code in the browser editing 
+
+Instead of editing in the browser using the GitHub user interface, you can also directly use
+the editor VSCode in the browser:
+
+- In [github.com/cphbiosciencephd/info..._pages/bikes.md)](https://github.com/cphbiosciencephd/info/blob/master/_pages/bikes.md) exchange the `.com` for `.dev`, i.e. [github.dev/cphbiosciencephd/info..._pages/bikes.md)](https://github.dev/cphbiosciencephd/info/blob/master/_pages/bikes.md) 
```

Comment:
There's an extra `)` in both links.

Reply:
thanks for spotting this:) You can also directly edit it in the form:)